exercises/rna-transcription/tests/rna-transcription.rs

```diff
@@ -9,29 +9,47 @@ fn test_acid_equals_acid() {
 #[test]
 #[ignore]
 fn test_transcribes_cytosine_guanine() {
-    assert_eq!(dna::RibonucleicAcid::new("G"), dna::DeoxyribonucleicAcid::new("C").to_rna());
+    assert_eq!(Ok(dna::RibonucleicAcid::new("G")), dna::DeoxyribonucleicAcid::new("C").to_rna());
```

Comment:
Well, this is a different approach from, say, #238 which used `unwrap`. Note that https://github.com/exercism/xrust/blob/master/exercises/nucleotide-codons/tests/codons.rs does use `assert_eq` with `Ok` though. I think I like this better. The difference is what happens when I return an `Err` when I should return `Ok`. With `assert_eq!(Ok(...), ...)`:

```
---- test_transcribes_cytosine_guanine stdout ----
	thread 'test_transcribes_cytosine_guanine' panicked at 'assertion failed: `(left == right)` (left: `Ok(RibonucleicAcid { nucleotides: "G" })`, right: `Err(())`)', tests/rna-transcription.rs:12
```

With `assert_eq!(..., ....unwrap())`;

```
---- test_transcribes_cytosine_guanine stdout ----
	thread 'test_transcribes_cytosine_guanine' panicked at 'called `Result::unwrap()` on an `Err` value: ()', src/libcore/result.rs:859
```

So why don't I go ahead and say now that I like `assert_eq(Ok(...), ...)` better since it shows me both sides.